packages/nc-gui/components/smartsheet/toolbar/FieldListAutoCompleteDropdown.vue

```diff
@@ -26,9 +26,11 @@ const options = computed<SelectProps['options']>(() =>
         return !(
           c.uidt === UITypes.LinkToAnotherRecord && (c.colOptions as LinkToAnotherRecordType).type !== RelationTypes.BELONGS_TO
```

Comment:
QrCode should be ignored in Sort as well.

![image](https://user-images.githubusercontent.com/35857179/202607943-cdb068e1-9250-421f-a3e4-0a4b8d3316ff.png)